CMakeLists.txt

```diff
@@ -51,6 +54,29 @@ if(${SIMULATOR} STREQUAL "Vicuna") # Vicuna only supports a reduced set of exten
   endif()
   set(RISCV_ABI "ilp32" CACHE STRING "mabi argument to the compiler")
   set(RISCV_CMODEL "medany" CACHE STRING "mcmodel argument to the compiler")
+###
+elseif(${TOOLCHAIN} STREQUAL "K230") # K230 hardware only supports vector extension, is 64 bit, and needs different ABI
+  if(USE_VEXT)
+    set(RISCV_ARCH "rv64gcv" CACHE STRING "march argument to the compiler, using the vector extension")
+  elseif(USE_PEXT)
+    message(FATAL_ERROR "K230 hardware does not support packed instructions.")
+  else()
+    set(RISCV_ARCH "rv64gc" CACHE STRING "march argument to the compiler, using only scalar instructions")
+  endif()
+  set(RISCV_ABI "lp64d" CACHE STRING "mabi argument to the compiler")
+  set(RISCV_CMODEL "medany" CACHE STRING "mcmodel argument to the compiler")
+
+elseif(${TOOLCHAIN} STREQUAL "K230_linux")
+  if(USE_VEXT)
+    set(RISCV_ARCH "rv64gcv" CACHE STRING "march argument to the compiler, using the vector extension")
+  elseif(USE_PEXT)
+    message(FATAL_ERROR "K230 hardware does not support packed instructions.")
+  else()
+    set(RISCV_ARCH "rv64gc" CACHE STRING "march argument to the compiler, using only scalar instructions")
+  endif()
+  set(RISCV_ABI "lp64d" CACHE STRING "mabi argument to the compiler")
+  set(RISCV_CMODEL "medlow" CACHE STRING "mcmodel argument to the compiler")
```

Comment:
Yes, that shouldn't be a problem. I only used `medlow` because all other non K230 toolchains use it. But having it consistent for both K230 toolchains makes sense.